WooCommerce/Classes/Extensions/NSAttributedString+Helpers.swift

```diff
@@ -10,32 +10,4 @@ extension NSAttributedString {
     var foundationString: NSString {
         return string as NSString
     }
-
-    /// Returns a copy of the receiver, with the Leading + Trailing newlines trimmed.
-    ///
-    func trimNewlines() -> NSAttributedString {
```

Comment:
Was this function no longer used?

Reply:
this function used to an internal function in `WordPressKit-iOS`, and it was recently made public for usage in the host app: https://github.com/wordpress-mobile/WordPressKit-iOS/commit/5505e41b5133f463e0156423d4c870321be7a068#diff-e07293832952daad30d7515bb8fe5823 and we got this update from the `WordPressAuthenticator` pod update (https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/blob/develop/WordPressAuthenticator.podspec#L42)  - the implementation we had looks like a cloned version of the one in `WordPressKit-iOS` 🙂